### ISSUE TITLE
[Netwrix_PAM] New integration, Netwrix Privilege Access Management

### ISSUE THREAD

**Issue by nicpenning** [elastic/integrations#11856]
There is no integration for Netwrix PAM solution. It would be great to have these events in Elastic.

**Comment by bhapas** [comment]
cc: @jamiehynds

**Comment by nicpenning** [comment]
https://community.netwrix.com/t/official-elastic-integration/3198

**Comment by cpascale43** [comment]
We are hoping to build this soon but still working through some partner paperwork, will keep this ticket updated. In the meantime @nicpenning wondering if you'd be able to share some sample data to get things off the ground? That would definitely help us move on this - can I send you a message in the Elastic Community slack?

**Comment by nicpenning** [comment]
Yes, apologies for the late response, I lost this in my notifications. Still would be nice to have this. Since this posting it seems Netwrix has more SIEM integration capabilities.

**Comment by nicpenning** [comment]
+1 as this still doesn't exist yet.
